docs/first-order/modality_alignment.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>So this plot is great. It does show the traces on top of each other. </p><p> </p><p>However, with matplotlib, you could have done this without the temporal alignment at all. </p><p>I think we need to add more analysis that makes use of the temporal alignment. </p><p>I suggest doing some stimuli triggered averages across all modalities. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>